routes/index.js

```diff
@@ -2,11 +2,13 @@ var path = require('path');
 var express = require('express');
 var morgan = require('morgan');
 var bodyParser = require('body-parser');
-var twilio = require('twilio');
-var config = require("../config");
+const Twilio = require('twilio');
+const config = require('../config');
+
 
 // Create a Twilio REST API client for authenticated requests to Twilio
-var client = twilio(config.accountSid, config.authToken);
+const client = new Twilio(config.accountSid, config.authToken);
```

Comment:
To address this one please refer to https://github.com/TwilioDevEd/marketing-notifications-node/pull/6/files#r87578365